pyproject.toml

```diff
@@ -12,22 +12,6 @@ exclude_dirs = ["tests"]
 line-length = 120
 target-version = ["py38"]
 
-[tool.coverage.paths]
-tests = ["tests"]
-vizro = ["src/vizro"]
```

Comment:
Is there a reason why we move this to the pyproject.toml of vizro-core instead of adapting the path here? So e.g. `vizro-core= ["vizro-core/src/vizro"]`. I thought the advantage of having it here is that we then configure the test coverage requirements for all sub-packages here

Reply:
But the test coverage requirements are not not global, but rather per package. Some of the settings could potentially act globally, but some definitely cannot. E.g. the `fail_under` should be done per package.

Reply:
Ah, so we cannot enforce a min. test coverage of 80% for all our sub-packages in the same pyproject.toml?

Reply:
I think in principle this could maybe be possible, but I think it would be better not too.

Say `vizro-core` is really mature, and we have a minimum coverage of 95%, but then there is a new package with 60% (which is fine to start with maybe in that case), what do we do.

The current idea is to always increase the minimal coverage when we test more, and just ensure it doesn't get lower after that